### DISCUSSION TITLE
add a quick recharge without the cursor and mouse

### DISCUSSION THREAD

**Discussion by gnomesh** [FakeFishGames/Barotrauma#7140] [Ideas and suggestions]
it would be nice if you added the possibility of reloading by double-clicking on the NUMBER on which the cartridge is located, if the weapon is in your hands

**Comment by StruggleDuck** [comment]
another possible option I just came up with is that there could be some button to reload your gun automatically (with a small half second delay), as long as there is suitable ammunition in a bandolier. This would increase the viability of the bandolier over the body armor, making it useful for security or captains that are not going around with an assault rifle or automatic shotgun.
